server/odc-service/src/main/java/com/oceanbase/odc/metadb/regulation/ruleset/RuleApplyingRepository.java

```diff
@@ -43,5 +47,30 @@ Optional<RuleApplyingEntity> findByOrganizationIdAndRulesetIdAndRuleMetadataId(L
     List<RuleApplyingEntity> findByOrganizationIdAndRuleMetadataName(@Param("organizationId") Long organizationId,
             @Param("name") String name);
 
+    @Transactional
+    int deleteByOrganizationIdAndRulesetId(Long organizationId, Long rulesetId);
+
+
+    default List<RuleApplyingEntity> batchCreate(List<RuleApplyingEntity> entities) {
+        String sql = InsertSqlTemplateBuilder.from("regulation_rule_applying")
+                .field(RuleApplyingEntity_.ORGANIZATION_ID)
+                .field(RuleApplyingEntity_.RULESET_ID)
+                .field(RuleApplyingEntity_.RULE_METADATA_ID)
+                .field(RuleApplyingEntity_.APPLIED_DIALECT_TYPES)
+                .field(RuleApplyingEntity_.ENABLED)
```

Comment:
The value of  `RuleApplyingEntity_.ENABLED` is "enabled", but the actual column name is "is_enabled". Have you ever test this case?

Reply:
You don't need to care about what the actual column name in tables, you only need to focus on the entity itself.

Reply:
Thanks for your careful review! Yes you are right, we should use `is_enabled` here. Plus, I added unit test cases for this method.

Reply:
😘